how-to/workspace-platform-starter/public/manifest.fin.json

```diff
@@ -447,6 +447,42 @@
 						}
 					}
 				},
+				{
+					"include": true,
+					"id": "include-in-snapshot",
+					"button": {
+						"type": "Custom",
+						"tooltip": "Include in Snapshot",
+						"disabled": false,
+						"iconUrl": "http://localhost:8080/icons/{scheme}/snapshot-removed.svg",
+						"action": {
+							"id": "include-in-snapshot",
+							"customData": {
+								"sourceId": "include-in-snapshot",
+								"replacementId": "remove-from-snapshot"
+							}
+						}
+					},
+					"conditions": ["removed-from-snapshot"]
+				},
+				{
+					"include": true,
```

Comment:
shouldn't this be false. i.e. don't include button?

Reply:
If the condition module is not enabled it won't show anyway, but I could set it to false.